scripts/rootpy-add-include-path

```diff
@@ -0,0 +1,23 @@
+#! /bin/env python
```

Comment:
Also, if no paths are given (``nargs='*'``) then you could print the current paths in use.